backend/hasura-actions/src/Scheduler.res

```diff
@@ -183,6 +194,57 @@ let startProcess = () => {
     },
     (),
   )
-
   start(job)
 } /* execute micropayment amount to recipientAddress [POST request with parameters */
+
+/*
+KEPT THIS AS A REMINDER FOR THE DEPOSIT
+let dummyData = [
+  {
+    recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
+    addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
+    lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
+    interval: 60, // this will always be 60 for our demo
+    // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
+    rate: "1",
+    deposit: "7200",
+    numerOfPaymentsMade: 0,
+    totalNumberOfPaymentsToMake: 1440,
+  },
+  {
+    recipient: "0xc788F08a2aAf539111e2a2D85BD4B324FBE37B15",
+    addressTokenStream: "0xb38981469B7235c42DDa836295bE8825Eb4A6389",
+    lengthOfPayment: 86400, // seconds [86400 equals one day.] Must be a multiple of 60
+    interval: 60, // this will always be 60 for our demo
+    // TODO: these values should be BigInt and use `@decco.codec` as the conversion function
+    rate: "1",
+    deposit: "14400",
+    numerOfPaymentsMade: 0,
+    totalNumberOfPaymentsToMake: 1440,
+  },
+|];
+*/
+
+/*
+curl -X POST --data-raw '{"amount": "100000000000000000"}' http://localhost:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/0x91c0c7b5D42e9B65C8071FbDeC7b1EC54D92AD92
+
+--data-raw '{"amount":"10000000000000000000","identifier":"1612189154951"}'
+
+curl 'http://localhost:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/0x91c0c7b5D42e9B65C8071FbDeC7b1EC54D92AD92' -H 'User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:79.0) Gecko/20100101 Firefox/79.0' -H 'Accept: application/json, text/plain, *' -H 'Accept-Language: en-US,en;q=0.5' --compressed -H 'Content-Type: application/json' -H 'Origin: http://localhost:5001' -H 'Connection: keep-alive' -H 'Referer: http://localhost:5001/ui/home' --data-raw '{"amount":"10000000000000000000","identifier":"1612189154951"}'
+
+curl 'http://localhost:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/0x91c0c7b5D42e9B65C8071FbDeC7b1EC54D92AD92' -H 'Content-Type: application/json' --data-raw '{"amount":"100000000000000000","identifier":"86"}'
+
+curl 'http://localhost:5001/api/v1/payments/0xC563388e2e2fdD422166eD5E76971D11eD37A466/0x6533180A035401d917535d889EfB8CE47Cf3bb22' -H 'Content-Type: application/json' --data '{"amount":"100000000000000000","identifier":"86"}'
+*/
+
+/*
+USEFUL INFO
+TokenAddress: 0xC563388e2e2fdD422166eD5E76971D11eD37A466
+RecipientAddress: 0x91c0c7b5D42e9B65C8071FbDeC7b1EC54D92AD92
+Amount = 100000000000000000
+*/
+
+/*
+nextPaymentTimestamp = 120, currentPaymentTimestamp = 600, interval = 2
+600 - 120 = 480 / 2 = 240 / 60 = 4
+*/
```

Comment:
I wonder if there is a better place to put these kinds of notes, it can quickly get messy and outdated having stuff like this in the source code.

Maybe a folder called `/docs/dev/notes.md` or something similar?